drink/src/session.rs

```diff
@@ -116,21 +111,21 @@ pub const NO_ARGS: &[String] = &[];
 ///     .deploy_bundle_and(contract, "new", NO_ARGS, vec![], None); /* ... */
 ///  # Ok(()) }
 /// ```
-pub struct Session<R: Runtime> {
+pub struct Session<R: Runtime + pallet_contracts::Config> {
```

Comment:
I think it might be a convenient thing to introduce a new trait like `RuntimeWithContracts`, which would just capture this generic bound (it's already used in a couple of places)